haiku-apps/stopwatch/stopwatch-1.0.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="A simple stopwatch application"
+DESCRIPTION="This stopwatch is very simple to use and allows a user to start, stop and reset the clock."
+HOMEPAGE="https://github.com/dcatt-haiku/StopWatch"
+COPYRIGHT="2016 DCatt"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dcatt-haiku/StopWatch/archive/v1.0.0.tar.gz"
+CHECKSUM_SHA256="320f0d7502316e12209c73576cacf488f49abd58a42cc986a84882a54eb9e732"
+SOURCE_DIR="StopWatch-1.0.0"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	StopWatch = $portVersion
```

Comment:
I just did, and (with that last change "cp -af objects/StopWatch $appsDir"), it does build.
So, the problem seems to be with your configuration. What's the output of "findpaths -r "makefile_engine" B_FIND_PATH_DEVELOP_DIRECTORY"? It should be "/boot/system/develop". If it isn't maybe that package isn't correctly installed after all. Try "pkgman search makefile_engine" to see if it's installed (then it's marked with an "s"), and maybe reinstall...